src/runtime_src/core/pcie/windows/core_system.cpp

```diff
@@ -32,11 +33,77 @@ void xrt_core::system::get_xrt_info(boost::property_tree::ptree &_pt)
 }
 
 
-void xrt_core::system::get_os_info(boost::property_tree::ptree &_pt)
+//function for converting CString type to std::string
+std::string convert(CString input)
 {
-  // TODO
-  _pt.put("windows", "");
+  CT2CA converted_input(input);
+  std::string string_input(converted_input);
+
+  return string_input;
 }
 
 
+CString GetStringFromReg(HKEY keyParent, CString keyName, CString keyValName)
```

Comment:
Return type should be: std::string
Function should be static
Parameters should be std::string instead of CString

For this helper method, it should a std::string for this is the string type that is needed.